docs/input.md

```diff
@@ -57,6 +57,16 @@ Inside `package:flame/gestures.dart` you can find a whole set of `mixin`s which
   - onScaleStart
   - onScaleUpdate
   - onScaleEnd
+
+
+ - MultiTouchTapDetector
+  - onTap
```

Comment:
are these names conflicting with the single tap mixin on purpose because you can only use one or the other? similar to that horizontal/vertical thing?

Reply:
You can't use the tap detector and multi tap detector together, and their signature are different, as the multi tap detector includes the pointer id.

I haven't added the horizontal/vertical drag on this PR, just the drag.